### DISCUSSION TITLE
How do I know if my current IDE is Positron or RStudio

### DISCUSSION THREAD

**Discussion by chuxinyuan** [posit-dev/positron#3826] [Q&A] [answered]
I am currently using Positron, but neither `devtools::session_info()` nor `xfun::session_info()` are currently giving me the correct IDE information.

**Comment by jennybc** [comment, marked as answer]
I'm sure the landscape will change, but at this moment the `POSITRON` environment variable is probably your best bet:

``` r
is_positron <- function() {
  Sys.getenv("POSITRON") == "1"
}

is_positron()
#> [1] TRUE
```

<sup>Created on 2024-07-02 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Reply by chuxinyuan** [reply]
Timely rain, thank you @jennybc
The version number information seems to be missing at the moment.

**Reply by chuxinyuan** [reply]
`Sys.getenv("POSITRON_VERSION")` can get the version number of Positron.
